sky/serve/serve_utils.py

```diff
@@ -636,9 +636,7 @@ def _get_replicas(service_record: Dict[str, Any]) -> str:
     for info in service_record['replica_info']:
         if info['status'] == serve_state.ReplicaStatus.READY:
             ready_replica_num += 1
-        # If auto restart enabled, not count FAILED replicas here.
-        if (not service_record['auto_restart'] or
-                info['status'] != serve_state.ReplicaStatus.FAILED):
+        if info['status'] != serve_state.ReplicaStatus.FAILED:
```

Comment:
Should we add a column in `sky serve status` for how many failed replicas we have? or left a todo here

Reply:
I think this is probably optional. Left a todo.